src/s1reader/s1_reader.py

```diff
@@ -349,29 +374,46 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     # burst sensing starts in prototyping test data
     burst_interval = 2.758277
 
-    #parse manifest.safe to retrieve IPF version
+    # parse manifest.safe to retrieve IPF version
     manifest_path = os.path.dirname(annotation_path).replace('annotation','') + 'manifest.safe'
     with open_method(manifest_path, 'r') as f_manifest:
         tree_manfest = ET.parse(f_manifest)
         ipf_version = get_ipf_version(tree_manfest)
 
-    #Load the Product annotation - for EAP calibration
-    #with open_method(annotation_path, 'r') as f_lads:
-    #    tree_lads = ET.parse(f_lads)
-    #    product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
+    # Load the Product annotation - for EAP calibration
+    with open_method(annotation_path, 'r') as f_lads:
+        tree_lads = ET.parse(f_lads)
+        product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
```

Comment:
```suggestion
        product_annotation = ProductAnnotation.from_et(tree_lads)
```
What do you think about changing imports up top so `*Annotation` doesn't need to be accessed by `s1_annotation`. The changes up top would be something like:
```
import s1_annotation # to access __file__
from s1_annotation import NoiseAnnotation, ProductionAnnotation
```

Reply:
I think that's helpful in simplifying the code, and improve the readability.
I've also imported to the classes below:
```python
ProductAnnotation
NoiseAnnotation
CalibrationAnnotation
AuxCal
BurstCalibration
BurstEAP
BurstNoise
```